samples/bluetooth/nrf_dm/src/pwm_led.c

```diff
@@ -50,5 +33,5 @@ void pwm_led_set(uint16_t desired_lvl)
 {
 	uint32_t scaled_lvl = (PWM_PERIOD * desired_lvl) / LIGHTNESS_MAX;
```

Comment:
Is `PWM_PERIOD` aligned with the DTS?

Reply:
I replaced the call below with `pwm_set_dt` and used `PWM_PERIOD` there.